spot_driver/include/spot_driver/api/default_state_client.hpp

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2024 Boston Dynamics AI Institute LLC. All rights reserved.
+
+#pragma once
+
+#include <bosdyn/client/robot_state/robot_state_client.h>
+#include <spot_driver/api/state_client_interface.hpp>
+#include <spot_driver/api/time_sync_api.hpp>
+
+#include <memory>
```

Comment:
In general the reason these includes are here is because the cpplint pre-commit step for this repo has an include-what-you-use rule, which isn't satisfied by having the header included by the base class.

Reply:
This example (and several others you pointed out) can be removed, though.